database/scripts/seed.js

```diff
@@ -14,51 +13,61 @@ mongoose.connect('mongodb://localhost/bonuses', {
   console.log('DB connection error:', err);
 });
 
-const soundtrackSeeds = () => {
```

Comment:
Removed soundtrack seeding from a function making albums global and put the songs generation in it's own helper function to facilitate the random addition of another album to some bonuses.